spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin2/sender/ZipkinRestTemplateSenderConfiguration.java

```diff
@@ -145,6 +147,8 @@ ZipkinLoadBalancer noOpLoadBalancer(final ZipkinProperties zipkinProperties) {
 
 	private static final Log log = LogFactory.getLog(ZipkinRestTemplateWrapper.class);
 
+	private static final int DEFAULT_TIMEOUT = 500;
```

Comment:
Is this different than before? I wonder if users in slow networks will start to get timeouts because of this.